libraries/helpers.rb

```diff
@@ -69,13 +79,17 @@ def jks_opts(prefix)
     end
 
     # Return hash with SSL options for OpenSSL
-    def ssl_opts
+    def cdap_ssl_opts(prefix = 'dashboard')
       ssl = {}
-      ssl['keypath'] = node['cdap']['cdap_security']['dashboard.ssl.key']
-      ssl['certpath'] = node['cdap']['cdap_security']['dashboard.ssl.cert']
+      ssl['keypath'] = node['cdap']['cdap_security']["#{prefix}.ssl.key"]
+      ssl['certpath'] = node['cdap']['cdap_security']["#{prefix}.ssl.cert"]
       ssl['common_name'] = node['cdap']['security']['ssl_common_name']
       ssl
     end
+
+    def cdap_property(property, sitefile = 'cdap_site')
```

Comment:
it comes down to what is the point of the helper.  I know the locations are not interchangeable, but the caller of the helper (3rd-party cookbook author) shouldn't have to.  We should validate separately from the helper IMO... but this is long-term concern.